### PR TITLE
feat : add image and input

### PR DESCRIPTION
**Pull request by ArjunJunna**
have added the image and input components.

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.input > input {
+  width: 25rem;
+  height: 4rem;
+  padding: 0.2em;
+  border-radius: 0.5rem;
+}
+
+.input > div {
+  position: absolute;
+  top: -0.6em;
```
Comment (ApurvaSawant11) [inline components/input/input.css:17]: Try not to use this selector. Add a separate class instead.

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  font-size: 0.88rem;
+  left: 1em;
+  background-color: var(--primary-color);
+  font-weight: 500;
+  padding: 0.1rem;
+}
+
```
Comment (Maianki) [inline components/input/input.css]: Avoid using child selectors, for better code maintainability.

**Review by Maianki — CHANGES_REQUESTED** [review]
Suggested a small change. Everything else looks good.

**Review by ApurvaSawant11 — COMMENTED** [review]
Also, a suggestion, add same styling to the textarea as that of input fields. Have uniformity in styling.

File: components/input/input.css (hunk 1 above)
Comment (ArjunJunna) [reply components/input/input.css:17]: Used separate classes now. Thank you.

File: components/input/input.css (hunk 2 above)
Comment (ArjunJunna) [reply components/input/input.css]: Removed them and added separate classes instead. Thank you.

**Comment by ArjunJunna** [comment]
> Also, a suggestion, add same styling to the textarea as that of input fields. Have uniformity in styling.

Done. Will maintain uniformity in styling.

**Pull request merged**